tsconfig.json

```diff
@@ -15,6 +15,8 @@
         "sourceMap": true
     },
     "include": [
+        "custom_typings/*.d.ts",
+        "node_modules/**/*.d.ts",
```

Comment:
We shouldn't include all .d.ts file in node_modules. tsc will automatically look in `@types` and in packages that are imported (looking for the "typings" field").